### ISSUE TITLE
stage2_with_object365.sh

### ISSUE THREAD

**Issue by lhqxcc** [SY-Xuan/Pink#17]
Hello, I encountered an issue while reproducing your work. I used your stage2_with_object365.sh file for training, but found that the batch size is very small, which leads to a very long training time. Moreover, the loss keeps increasing during the training process. Could you please tell me what might be causing this problem?

<img width="817" alt="Image" src="https://github.com/user-attachments/assets/5a06c281-bd43-47e4-817c-a95e1c3c4322" />

**Comment by SY-Xuan** [comment]
Do you modify the batch_size? How many GPUs do you use? Do you load the weight of stage1?

**Comment by lhqxcc** [comment]
Thank you very much for your response. Below is my training script. Please keep the script you provided unchanged: 

![Image](https://github.com/user-attachments/assets/efe33349-f0e7-43cf-9577-d6424aa83861)
I found the problem in the training log file:

![Image](https://github.com/user-attachments/assets/37a53a6c-daee-447f-9793-05589b4ea08c)

What do you think might have caused this problem?

**Comment by SY-Xuan** [comment]
Maybe you can try a learning rate=2.5e-4. Sometimes it may have a loss spike.
